### PR TITLE
ZCS-1936 [hi, ca, no, ko] - tinyMCE strings are shown in English

### PR DESCRIPTION
**Pull request by silentsakky**
- match tinymce locale name with zimbra locales

### CODE REVIEW

File: tiny_mce/custom/langs/ca.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * All portions of the code are Copyright (C) 2014, 2016 Synacor, Inc. All Rights Reserved.
  * ***** END LICENSE BLOCK *****
  */
-tinymce.addI18n('ca-ES',{
+tinymce.addI18n('ca',{
```
Comment (SatyamKumarOnline) [inline tiny_mce/custom/langs/ca.js]: Double quotes, to be at consistent with others.

File: tiny_mce/custom/langs/hi.js [hunk 2]
```diff
@@ -20,7 +20,7 @@
  * All portions of the code are Copyright (C) 2016 Synacor, Inc. All Rights Reserved.
  * ***** END LICENSE BLOCK *****
  */
-tinymce.addI18n('hi_IN',{
+tinymce.addI18n('hi',{
```
Comment (SatyamKumarOnline) [inline tiny_mce/custom/langs/hi.js]: Same here and below changes.

File: tiny_mce/custom/langs/ca.js (hunk 1 above)
Comment (silentsakky) [inline tiny_mce/custom/langs/ca.js]: Fixed in all files

File: tiny_mce/custom/langs/hi.js (hunk 2 above)
Comment (silentsakky) [inline tiny_mce/custom/langs/hi.js]: Fixed in all files

**Pull request merged**